From 34add045fd2ec3f1031dee961c4d9cc4285486a7 Mon Sep 17 00:00:00 2001
From: Sun Haiyong <youbest@sina.com>
Date: Tue, 31 Aug 2021 11:11:52 +0800
Subject: [PATCH] stack-direction: Add support for loongarch CPU

* m4/stack-direction.m4 (SV_STACK_DIRECTION): When the CPU is loongarch,
set "sv_cv_stack_direction" to "-1" .
---
 m4/stack-direction.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index 93287254b..e6c3284ce 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -32,6 +32,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       i?86 | x86_64 | \
       i860 | \
       ia64 | \
+      loongarch* | \
       m32r | \
       m68* | \
       m88k | \
-- 
2.17.2

